### PR TITLE
Regenerated pelorus-operator with new operator-sdk version

### PR DESCRIPTION
**Pull request by mpryc**
Updated operator with new operator-sdk version 1.33.

Please go the the `Preview` tab and select the appropriate template for your PR:

* [General PR](?expand=1&template=general_template.md)
* [Release PR](?expand=1&template=release_template.md)

### CODE REVIEW

File: pelorus-operator/Makefile.orig [hunk 1]
```diff
@@ -0,0 +1,227 @@
+# VERSION defines the project version for the bundle.
```
Comment (etsauer) [inline pelorus-operator/Makefile.orig:1]: We don't need this.

Comment (mpryc) [reply pelorus-operator/Makefile.orig]: Removed.

File: pelorus-operator/config/rbac/role.yaml [hunk 2]
```diff
@@ -1,106 +1,15 @@
...
+  labels:
+    app.kubernetes.io/name: clusterrole
+    app.kubernetes.io/instance: manager-role
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: pelorus-operator
+    app.kubernetes.io/part-of: pelorus-operator
+    app.kubernetes.io/managed-by: kustomize
   name: manager-role
 rules:
-##
```
Comment (mateusoliveira43) [inline pelorus-operator/config/rbac/role.yaml:6]: is this removal of permissions ok :question:

Comment (mpryc) [reply pelorus-operator/config/rbac/role.yaml:6]: TBH I don't know as this is generated by the operator-sdk, so we need to test that operator to see.

**Review by etsauer — APPROVED** [review]
Tested the operator deployment, and it looks good.

**Review by mateusoliveira43 — APPROVED** [review]
did not test, but code looks good to me

**Pull request merged**